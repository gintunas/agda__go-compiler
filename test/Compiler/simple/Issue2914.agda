
open import Agda.Builtin.IO
open import Agda.Builtin.Nat
open import Agda.Builtin.Unit

private
  n : Nat
  n = 7

{-# COMPILE GHC n as n #-}

postulate
  main : IO ‚ä§

{-# COMPILE GHC main = print n #-}
