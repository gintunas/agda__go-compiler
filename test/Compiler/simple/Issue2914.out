EXECUTED_PROGRAM

ret > ExitSuccess
out > 7
out >
